### PR TITLE
Image fixes and Team Updates

### PR DESCRIPTION
**Pull request by jonathankwhitmer**
This addresses some of the concerns I mentioned on Slack Wednesday. I'm currently updating the publication list as well, but that's a bit slower since it was last updated on the site two years ago and many new papers have come out since then.

### CODE REVIEW

**Comment by pabloferz** [comment]
I forgot to mention, the `index.css` covers some aspects of the elements of the main page, but other properties of elements that apply more broadly to the site are defined in `main.css`.

`main.css` has already the `img` properties, so the ones here should be moved there, and to make it consistent we should put the `video` selector also there.

File: assets/css/index.css [hunk 1]
```diff
@@ -2,6 +2,24 @@ body {
...
 
+img {
+
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 50px;
+  padding-bottom: 50px;
+
+}
```
Comment (mquevill) [inline assets/css/index.css:13]:
```suggestion
img {
  display: block;
  margin-left: auto;
  margin-right: auto;
  padding-top: 50px;
  padding-bottom: 50px;
}
```

File: team/index.html [hunk 2]
```diff
@@ -274,29 +274,42 @@ <h1 id="team-members">Meet the Team</h1>
...
+<h2 style="line-height:3;">Current SSAGES Development Team</h2>
 
-<p>Those who have contributed code, with a brief summary of their contributions
-include:</p>
+<p>Active contributors, with a brief summary of their contributions:</p>
+
+<p>
+  <ul style="padding-left: 30px;">
+    <li>Michael Quevillon (Whitmer Group), core functionality, collective variables</li>
+    <li>Pablo Zubieta (de Pablo Group), core functionality</li>
```
Comment (mquevill) [inline team/index.html:284]:
```suggestion
    <li>Michael Quevillon (Whitmer Group), core functionality, collective variables, methods</li>
    <li>Pablo Zubieta (de Pablo Group), core functionality, methods</li>
```

**Review by mquevill — CHANGES_REQUESTED** [review]
I agree that the `img` and `video` definitions should be moved to `main.css`.

File: team/index.html [hunk 3]
```diff
@@ -274,29 +274,42 @@ <h1 id="team-members">Meet the Team</h1>
 
 <hr />
 
-<h2 style="line-height:3;">SSAGES Core Developers</h2>
+<h2 style="line-height:3;">Current SSAGES Development Team</h2>
```
Comment (pabloferz) [inline team/index.html:277]: Maybe we can leave just one h2 header `SSAGES Development Team` and separate with subheaders or `<p>` block as here, active and past contributors, removing the `Past Contributors` header below?

Comment (jonathankwhitmer) [reply team/index.html]: Done.

File: team/index.html (hunk 2 above)
Comment (jonathankwhitmer) [reply team/index.html:284]: Missed one piece here. Adding.

File: assets/css/index.css (hunk 1 above)
Comment (jonathankwhitmer) [reply assets/css/index.css]: This commit is obsolete since I moved the img construct to main.css. I did perform this change there.

**Review by mquevill — CHANGES_REQUESTED** [review]

**Review by mquevill — APPROVED** [review]

**Pull request merged**